editor.planx.uk/src/pages/FlowEditor/components/Flow/components/Node.tsx

```diff
@@ -145,6 +140,17 @@ const Node: React.FC<any> = (props) => {
   }
 };
 
+const getSetValueText = ({ operation, fn, val }: Store.node["data"]) => {
+  switch (operation) {
+    case "append":
+      return `Append ${val} to ${fn}`;
+    case "remove":
+      return `Remove ${fn}`;
```

Comment:
_nit_: 

It would be handy to have the `Remove ${val} from `${fn}` here if you have the chance.

Reply:
Good catch! See [367fd74](https://github.com/theopensystemslab/planx-new/pull/2914/commits/367fd74a0a8c2425005596632e1184c490c7dac8)